src/Makefile

```diff
@@ -79,9 +80,21 @@ $(OUTPUT)/retsnoop.skel.h: $(OUTPUT)/mass_attach.bpf.o
 $(OUTPUT)/retsnoop.o: $(OUTPUT)/retsnoop.skel.h $(OUTPUT)/calib_feat.skel.h
 $(OUTPUT)/mass_attacher.o: $(OUTPUT)/retsnoop.skel.h $(OUTPUT)/calib_feat.skel.h
 
-$(OUTPUT)/addr2line.embed.o: ../tools/addr2line
+../sidecar/target/release/addr2line::
+	$(call msg,CARGO,addr2line)
+	$(Q)if [ -e $@ ]; then touch $@.timestamp; fi
+	$(Q)cargo build --manifest-path=../sidecar/Cargo.toml --release
+	$(Q)if [ ! -e $@.timestamp -o $@ -nt $@.timestamp ]; then \
+		$(LLVM_STRIP) -g $@; \
+	fi
+
+$(OUTPUT)/addr2line.embed.o: ../sidecar/target/release/addr2line
 	$(call msg,LD,$@)
-	$(Q)$(LD) -r -b binary -o $@ $<
+	$(Q)$(LD) -r -b binary \
```

Comment:
so here we can

```
cp $< sidecar.tmp
strip -g sidecar.tmp
/* now embed sidecar.tmp */
```

Then just add *.tmp into .gitignore to not accidentally check it in. WDYT?